src/ORM/Query.php

```diff
@@ -379,12 +379,11 @@ public function contain($associations = null, $override = false)
     {
         $loader = $this->getEagerLoader();
         if ($override === true) {
-            $loader->clearContain();
-            $this->_dirty();
+            $this->clearContain();
         }
 
         if ($associations === null) {
-            return $loader->contain();
+            return $this->getEagerLoader()->contain();
```

Comment:
You don't need the `getEagerLoader()` call here, this method already has a `$loader` instance available.